updatecli/updatecli.d/update-deps.yaml

```diff
@@ -0,0 +1,138 @@
+name: Update charts with new policy versions, kubectl image
+
+sources:
+  kubectlImageTag:
+    kind: dockerimage
+    spec:
+      image: ghcr.io/kubewarden/kubectl
+      versionfilter:
+        kind: semver
```

Comment:
The test in my fork I can see the script bumping the version. That's nice! Thanks for that!

But I have some questions, how the `versionfilter` knows which is the latest version? Does it perform semver comparisons? What are the difference from the kind `latest`? 

I'm wondering if we can hit some coner case like a patch release is published after a major/minor release. In other words, let's image that we have a latest release which is `v2.0.0` but we release a patch release `v1.9.1` to fix an issue in the `v1.9.0` version. Will the script downgrade the policy because `v1.9.1` is the most recent version? I was reading the updatecli [docs](https://www.updatecli.io/docs/plugins/_versionfilter/) I did not fully understand what's happening here.

Reply:
I'm aware that scenario never happen before. But I would like to understand that to help the maintainance of the script in the future.

Reply:
Good question, tripped me too.
Right now the `spec.versionfilter` is configured with `semver` and no `pattern`. This means it defaults to `pattern: *`, and will accept any valid semver (including prereleases, which I think it's fine, we review the PR) that are newer than the current semver version (so, not `:latest` nor downgrades). See [here](https://www.updatecli.io/docs/plugins/_versionfilter/).